db/table_cache_sync_and_async.h

```diff
@@ -79,7 +79,7 @@ DEFINE_SYNC_AND_ASYNC(Status, TableCache::MultiGet)
         assert(t);
       }
     }
-    if (s.ok() && !options.ignore_range_deletions) {
+    if (s.ok() && !options.ignore_range_deletions && !skip_range_deletions) {
```

Comment:
Just for my understanding, what's the difference between `options.ignore_range_deletions` and `skip_range_deletions`?

Reply:
They mean the same in this context. I didn't want to use `options.ignore_range_deletions` as I was concerned about side effects (I don't think there are any now, but not sure if there'll be in the future if the option is used in the table reader for example). Also, didn't want the overhead of making a copy of ReadOptions just to set this option.